person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :age, :name
+
+  def initialize(age:, name: 'unknown', parent_permission: true)
+    @id = Random.rand(1..100)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_service?
+    of_age? && @parent_permission
```

Comment:
- [ ] We are trying to check if a person is of age **OR** has permission, not if they are of age **AND** have permission. For that, we should use

  ```
  of_age? || @parent_permission
  ```

Reply:
This issue is well resolved! ✔️